lib/vagrant-openshift/action/generate_template.rb

```diff
@@ -40,7 +40,7 @@ def call(env)
           box_info = box_info_data[os][stage]
           box_info[:instance_name] = @options[:name].nil? ? 'openshift_origin_' + inst_ts : @options[:name]
           box_info[:os] = os
-          box_info[:vagrant_guest] = [:centos7, :rhel7].include?(os) ? :redhat : os
+          box_info[:vagrant_guest] = [:centos7, :rhel7, :rhelatomic].include?(os) ? :redhat : os
```

Comment:
Ok I can make that change although we don't care about version number when it comes to atomic when we release it I believe.  It isn't like Centos7 or RHEL7 we just release an image of atomic and update it through ostree.